### PR TITLE
Fix Issue 6345: Fixed a bug where creating a page via the cms.api.create_page ignores left/right positions

### LINKED ISSUE #6345: add_sibling (and thus create_page) ignores "left" and "right" positions
### Summary

Positions "left" and "right" are not respected when creating a Node.

Example
```
import cms.api
parent = cms.api.create_page(title="parent", template="example.html", language="de")
c2 = cms.api.create_page(title="child 2", template="example.html", parent=parent, language="de")
c1 = cms.api.create_page(title="child 1", template="example.html", parent=c2,position="left", language="de")
list(p.get_title("de") for p in parent.get_child_pages())
```
Result
```
['child 2', 'child 1']
```

### Expected behaviour

Adding a djangocms page at a position "left" or "right" of another node using cms.api.create_page is added left/right of the specified page.

The example above should result in
```
['child 1', 'child 2']
```

### Actual behaviour

Adding a djangocms page at a position "left" or "right" of another node using cms.api.create_page is added as "last child" of parent.

### Environment

* Python version: 3.6.4
* Django version: 1.11.11
* django CMS version: 3.5.1

### Proposed Fix
Replace `return super(TreeNode, self).add_sibling(*args, **kwargs)` by `return super(TreeNode, self).add_sibling(*args, pos, **kwargs)` in `cms/models/pagemode.py`

### PR DESCRIPTION
**Pull request by moellering**
### Summary
Fixes #6345
Fixed a bug where creating a page via the cms.api.create_page ignores left/right positions.
During the last refactor of the pagemodel add_sibling function the position parameter got lost.

### Proposed changes in this pull request
Forward the parameter to the page node again.

### Documentation checklist

* [x] I have updated CHANGELOG.txt if appropriate
* [x] I have updated the release notes document if appropriate, with:
    * [x] general notes
    * [x] bug-fixes
    * [x] improvements/new features
    * [x] backwards-incompatible changes
    * [x] required upgrade steps
    * [x] names of contributors
* [x] I have updated other documentation
* [x] I have added my name to the AUTHORS file
* [ ] This PR's documentation has been approved by Daniele Procida

### CODE REVIEW

File: cms/tests/test_page.py [hunk 1]
```diff
@@ -501,6 +501,16 @@ def test_publish_page_regression_6188(self):
         # Now there should be "en", "de" and "fr" on the public page
         self.assertSequenceEqual(sorted(page.publisher_public.get_languages()), ['de', 'en', 'fr'])
 
+    def test_create_page_with_position_regression_6345(self):
+        # ref: https://github.com/divio/django-cms/issues/6345
+        parent = create_page("p", "nav_playground.html", "en")
+        rightmost = create_page("r", "nav_playground.html", "en",parent=parent)
```
Comment (vThaian) [inline cms/tests/test_page.py]: please add space here

File: cms/tests/test_page.py [hunk 2]
```diff
@@ -501,6 +501,16 @@ def test_publish_page_regression_6188(self):
...
+        leftmost = create_page("l", "nav_playground.html", "en",parent=rightmost, position="left")
```
Comment (vThaian) [inline cms/tests/test_page.py]: please add space here

File: cms/tests/test_page.py (hunk 1 above)
Comment (vThaian) [reply cms/tests/test_page.py]: please also move this test to test_api https://github.com/divio/django-cms/blob/develop/cms/tests/test_api.py

**Comment by FinalAngel** [comment]
thank you very much for the contribution @moellering, would you be able to adapt the PR according to the commends and set the merging branch to **develop**?

**Comment by FinalAngel** [comment]
Closed in favour of https://github.com/divio/django-cms/pull/6759, merged into develop. Thank you @moellering

**Pull request closed without merging**